client/MentorPanel/src/redux/actions.js

```diff
@@ -68,7 +68,7 @@ const loadTopics = (mentor_id, questions, recommended) => async (dispatch) => {
 
     if (recommended) {
       topic_questions = {
-        ['Recommended']: recommended,
+        ['Recommended']: recommended.constructor === Array ? recommended : [recommended],
```

Comment:
is this to check whether recommended is a js array object? can this just be `Array.isArray(recommended)` ?